### PR TITLE
Add `StepProgress` component

### LINKED ISSUE #7: Add a warning when user trying to set identity on account that already has one

### LINKED ISSUE #13: Implement a `StepProgress` component
With a similar look like one on Polygon Web Wallet

Screen Shot 2022-07-06 at 10 29 49

**Requirements**

- `StepProgress` should accept at least 2 props...
  + `steps: string[]` - array of all step labels
  + `stepIndex: number` - current step index

- When `stepIndex = 0`, the spinner is showing on step 0, and step 1 is highlighted
- When `stepIndex = 1`, the check-box is showing on step 0, and step 1 is showing spinner
- When `stepIndex = steps.length`, everything shows as completed with checkboxes

### LINKED ISSUE #23: Update `/identity/connect` page with correct content
## Set your identity
The Identity Module ensures an authentic governance and voting experience. It does this by requiring every voting wallet to be connected to 2 social accounts.

## Connect your wallet
Connect your voting wallet here. This is the wallet that will be checked against the staking requirement. If you have multiple staking wallets, it is best to set a controller wallet for all your staked funds and connect this to the Identity Module to maximise your voting power.

## Connect your socials
If your wallet is yet to be associated with a social account, you will be able to sign in to  Twitter and Discord below. This will establish that you are the owner of the social accounts and therefore a real individual. After seeing both the ‘verified Twitter’ and ‘verified Discord’ icons, sign and submit the transaction to send this information to the blockchain.

## Sign and submit
Your identity has been successfully set. Visit Discord to view the Governance channels with your new role!
---
Discord link https://discord.gg/zbwXQZCcwr

### PR DESCRIPTION
**Pull request by aidan-starke**
## Summary

 - Closes #7
 - Closes #13 
 - Closes #23

## Preview

https://app-gov-git-spike-step-progress-cennznet.vercel.app/

### CODE REVIEW

File: apps/gov-website/pages/identity/connect.tsx [hunk 1]
```diff
@@ -98,19 +97,18 @@ const Connect: NextPage<StaticProps> = ({
 						Set your identity
 					</h1>
 
-					<p className="mb-8 text-lg">
-						To become a Citizen or Councillor, we need you to verify your
-						identity. This involves connecting your wallet, and two social
-						channels (Twitter and Discord). Get started below!
+					<p className="prose mb-8 text-lg">
```
Comment (ken-futureverse) [inline apps/gov-website/pages/identity/connect.tsx:107]: nice `prose` class!

**Review by ken-futureverse — CHANGES_REQUESTED** [review]
Couple of styling things

 - [x] Instead of grey, use a lighter colour of the purple
 - [x] Use a solid checkbox, might look better

And one issue that I found

Screen Shot 2022-07-08 at 10 20 16

File: apps/gov-website/pages/identity/connect.tsx (hunk 1 above)
Comment (aidan-starke) [reply apps/gov-website/pages/identity/connect.tsx:107]: :sunglasses:

**Review by ken-futureverse — CHANGES_REQUESTED** [review]
Looks good, the dialog needs to tweaks to address couple issues, filed them in #31 and #32 

One thing that needs to fix in this PR is the warning here, not sure what causes it

Screen Shot 2022-07-10 at 17 30 35

File: libs/web/hooks/src/useIdentityConnectForm.ts [hunk 2]
```diff
@@ -8,15 +8,19 @@ import {
 import { useCENNZApi, useCENNZWallet } from "@app-gov/web/providers";
 
 export interface IdentityFormState {
-	step: "Await" | "Submit" | "Process";
+	step: "Await" | "Submit" | "Process" | "Idle";
```
Comment (ken-futureverse) [inline libs/web/hooks/src/useIdentityConnectForm.ts]:
```suggestion
	step: "Idle" | "Await" | "Submit" | "Process";
```

**Review by ken-futureverse — DISMISSED** [review]

**Review by ken-futureverse — APPROVED** [review]

**Pull request merged**